packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -384,6 +390,15 @@ export class LitNodeClientNodeJs
         `Storage key "${storageKey}" is missing. Not a problem. Contiune...`
       );
 
+      // Check if a valid session key exists in cache
+      if (
+        sessionKeyCache &&
+        Date.now() - sessionKeyCache.timestamp < expirationInMs
```

Comment:
can we use the timestamp from the `siwe` here instead of creating a new timestamp for the expiration?